### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/6fee4884a1a097c1c4e395950da8be9f131de9f6

### CODE REVIEW

File: en/privacy/cookie-settings.ftl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/privacy/websites/cookie-settings/
+
+cookie-settings-page-title = Cookie settings
+cookie-settings-page-desc = This page describes the different types of Cookies that { -brand-name-mozilla } may use, and gives you control over which types of data you agree to { -brand-name-mozilla } collecting.
+cookie-settings-breadcrumb-link = Previous page
+cookie-settings-explainer = Cookies are small files containing pieces of information that are saved to your computer or device when you visit a website. { -brand-name-mozilla } uses Cookies to help make our websites work, as well as to collect information on how you use and interact with our websites, such as the pages you visit.
+cookie-settings-page-intro = This page describes the different types of Cookies and similar technologies (such as pixel tags, web beacons, clear GIFs, JavaScript, and local storage) (“Cookies”) that { -brand-name-mozilla } may use, and gives you control over which types of data you agree to { -brand-name-mozilla } collecting.
```
Comment (peiying2) [inline en/privacy/cookie-settings.ftl:11]: The reference to `(“Cookies”)` seems at an odd place in this sentence. 

It either comes right after `... different types of Cookies ...` or not needed.

**Review by peiying2 — COMMENTED** [review]
Other than when referencing `Cookies` by itself, not a UI term, should it be lower case?

File: en/privacy/cookie-settings.ftl (hunk 1 above)
Comment (alexgibson) [reply en/privacy/cookie-settings.ftl:11]: This is copy directly from legal, so I’m not going to change how it is worded here.

**Comment by alexgibson** [comment]
@peiying2 updated thanks

**Review by peiying2 — APPROVED** [review]

**Pull request merged**